mzLib/FlashLFQ/PeakIndexingEngine/IFlashLfqIndexingEngine.cs

```diff
@@ -34,7 +34,7 @@ public interface IFlashLfqIndexingEngine
         /// <param name="missedScansAllowed"> the number of successive missed scans allowed before the xic is terminated </param>
         /// <param name="maxPeakHalfWidth"> the maximum distance from the apex RT of the XIC to both start RT and end RT </param>
         /// <returns> A list of IIndexedPeak objects, ordered by retention time </returns>
-        public List<IIndexedPeak> GetXic(double mz, double retentionTime, Tolerance ppmTolerance, int missedScansAllowed, double maxPeakHalfWidth = int.MaxValue);
+        public List<IIndexedPeak> GetXic(double mz, double retentionTime, Tolerance ppmTolerance, int missedScansAllowed, double maxPeakHalfWidth = int.MaxValue, Dictionary<IIndexedPeak, ExtractedIonChromatogram> matchedPeaks = null);
```

Comment:
Please edit the summary comment explaining what the new matchedPeaks parameter does